test/test_data/test_validation.py

```diff
@@ -80,6 +82,23 @@ def test_dataframe_input_categorical(self):
             self.assertIsNotNone(validator.target_encoder)
             self.assertIsNotNone(validator.feature_encoder)
 
+    def test_binary_conversion(self):
```

Comment:
Seeing this, do you think it would make sense to test all supported tasks, i.e. binary classification, multiclass classification, multilabel classification, regression and multi-output regression?